packages/mgt/README.md

```diff
@@ -53,15 +53,16 @@ You can use the components by referencing the loader directly (via unpkg), or in
 ### Use via mgt-loader:
 
 ```html
-<script src="https://unpkg.com/@microsoft/mgt@3/dist/bundle/mgt-loader.js"></script>
+<script src="https://unpkg.com/@microsoft/mgt-components@3/dist/bundle/mgt-loader.js"></script>
```

Comment:
`mgt-loader.js` is only in the `@microsoft/mgt` package. This will break applications

Reply:
I think the changes WRT resources in components and msal2-provider should be made. mgt-loader is a special case that includes a few other dependencies for direct usage in an html file outside of a build system or needing to have other modules imported too.